engine/src/main/java/org/camunda/bpm/engine/impl/pvm/runtime/PvmExecutionImpl.java

```diff
@@ -173,6 +173,14 @@ public abstract class PvmExecutionImpl extends CoreExecution implements
 
   protected boolean activityInstanceEndListenersFailed = false;
 
+  /**
+   * Controls weather the current {@link Context#getBpmnExecutionContext() context execution}
+   * will be updated when creating an event subscription for the current scope.
+   *
+   * @see #setUpdateContextExecution(boolean)
+   */
+  protected boolean updateContextExecution = false;
```

Comment:
I don't think we need this flag, because we want this behavior in all cases when the execution is initialized (process start, process instance modification when an activity is started in a new subprocess, a process instance started via call activity with an event subprocess in the called process). Please add some tests for these cases, too.  We can discuss if there is a problem with the other cases.